application/helpers/admin/htmleditor_helper.php

```diff
@@ -221,7 +221,8 @@ function getInlineEditor($fieldtype, $fieldname, $fieldtext, $surveyID = null, $
         $toolbaroption = "";
         $sFileBrowserAvailable = '';
         $htmlformatoption = "";
-        $oCKeditorVarName = "oCKeditor_".str_replace("-", "_", $fieldname);
+        $oCKeditorVarName = "oCKeditor_".preg_replace("/[-\[]/", "_", $fieldname);
```

Comment:
Would be nice with a comment here explaining why this is needed.

Reply:
Added it on another PR: https://github.com/LimeSurvey/LimeSurvey/pull/1701

Reply:
Sweet! :star2: